neurogym/core.py

```diff
@@ -254,6 +255,10 @@ def sample_time(self, period):
             elif dist == "until":
                 # set period duration such that self.t_end[period] = args
                 t = args - self.tmax
+                if t < 0:
+                    msg = f"""Invalid 'until' time for period {period}. Current max time: {self.tmax},
+                    Requested end time: {args}"""
+                    raise ValueError(msg)
```

Comment:
this avoids printing the newline and indents in the error message.

```suggestion
                    msg = (
                        f"Invalid 'until' time for period {period}. Current max time: {self.tmax},",
                        f"Requested end time: {args}",
                    )
                    raise ValueError(msg)
```